themes/default/content/docs/get-started/aws/begin.md

```diff
@@ -95,6 +95,6 @@ $ export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
 
 For additional information on setting and using AWS credentials, see [AWS Setup]({{< relref "/docs/intro/cloud-providers/aws/setup" >}}).
 
-Next, you'll create a new project.
+Next, you'll create a new Pulumi project.
```

Comment:
do we need both `create` and `new` in these items?

Reply:
We definitely don't _need_ the word new here, no. But IMO it's fine to leave it;  some people would say "create a new thing", others would say "create a thing". It's clear either way, though, so since someone wrote it, and there's nothing wrong with it, I'm happy to leave it it. (I'm also one of those people who _would_ say "create a new thing", since that's how I speak -- although I didn't write these words originally.)